lib/hashauth.js

```diff
@@ -74,16 +74,18 @@ hashauth.init = function init(client, $) {
     }
     return false;
   };
-  
+
   hashauth.requestAuthentication = function requestAuthentication (eventOrNext) {
     var translate = client.translate;
     hashauth.injectHtml();
     $( '#requestauthenticationdialog' ).dialog({
-        width: 350
-      , height: 240
+        width: 400
+      , height: 270
+      , closeText: ''
```

Comment:
Make a auth dialog a little bit bigger, because in other languages the text is longer and scroll bars appear.